src/main/java/frc/robot/Constants.java

```diff
@@ -34,12 +23,40 @@
 import frc.robot.subsystems.Elevator;
 
 public final class Constants {
+  public static class MotorIDConstants {
+    public static final int kCLimberMotorLeader = 12;
+    public static final int kCLimberMotorFollower = 13;
+    public static final int kScorerMotor = 9;
+    public static final int kElevatorMotor1 = 10;
+    public static final int kElevatorMotor2 = 11;
+    public static final int kIntakeMotor = 9;
+  }
```

Comment:
We should not leave these as conflicting IDs. Lets change the climber motors to 16 and 17, and Scorer should be 12, and intake has 13-15